README.md

```diff
@@ -72,6 +71,34 @@ For multi-speaker models, use `--speaker <number>` to change speakers (default:
 
 See `piper --help` for more options.
 
+## Installation and Usage with Docker
+
+You can build a Docker image:
+
+```sh
+docker buildx build --target build -t piper:latest .
+```
+
+After building, you can connect to the container, download voices into it and run `piper`:
+
+```sh
+docker run -it --entrypoint bash piper:latest
+
+# run within the container
+cd /dist/piper
+curl -L --output voice-en-us-lessac-medium.tar.gz https://github.com/rhasspy/piper/releases/download/v0.0.2/voice-en-us-lessac-medium.tar.gz
+tar -xvf voice-en-us-lessac-medium.tar.gz
+echo 'Welcome to the world of speech synthesis!' | \
+  ./piper --model en-us-lessac-medium.onnx --output_file welcome.wav
+
+# while the container is running, and in a different terminal, copy the output file to the host
+docker cp $(docker ps -lq):/dist/piper/welcome.wav .
```

Comment:
based on the comments suggesting that the `--output_file` and `--output_dir` command line arguments are not working as expected (could be working and putting it in the wrong path?), here's a possible workaround for the current directory:

```suggestion
echo 'Welcome to the world of speech synthesis!' | \
  ./piper --model en-us-lessac-medium.onnx

# while the container is running, and in a different terminal, copy the output files to the host
docker exec $(docker ps -lq)bash -c "ls /dist/piper/*.wav" | while read line; do docker cp $(docker ps -lq):/$line .; done
```

based on this [StackOverflow answer](https://stackoverflow.com/a/75590085), I have not tested this yet

Reply:
Kept running into errors with this along the lines of:
![image](https://github.com/rhasspy/piper/assets/47067154/49b52d59-fba3-4438-a6ae-a94137311804)

However I was able to get a quick bash script that accomplishes copying all of the files over:
```sh
#!/bin/bash

# Get the ID of the most recently created Docker container
container_id=$(docker ps -lq)

# Execute a command in the Docker container to get the list of .wav files
files=$(docker exec -it $container_id bash -c "cd /dist/piper && ls *.wav")

# Remove any carriage return characters (in case this script is run on Windows)
files=${files//$'\r'/}

# Loop over each file
for file in $files; do
  # Copy the file from the Docker container to the current directory
  docker cp "$container_id:/dist/piper/$file" .
done
```

Reply:
that script works; on Windows I had to run `bash` first

Reply:
sweet